config/initializers/new_framework_defaults_7_0.rb

```diff
@@ -50,15 +50,15 @@
   "-vf 'select=eq(n\\,0)+eq(key\\,1)+gt(scene\\,0.015),loop=loop=-1:size=2,trim=start_frame=1' -frames:v 1 -f image2"
 
 # Automatically infer `inverse_of` for associations with a scope.
-# Rails.application.config.active_record.automatic_scope_inversing = true
+Rails.application.config.active_record.automatic_scope_inversing = true
```

Comment:
@camckin10 It's hard to know that there's no place where we make an assumption about a scope and its inverse. Here's an [explanation](https://blog.kiprosh.com/rails-7-auto-detects-inverse-of-for-scoped-associations/) of the change. I think it's likely we're fine but we should be careful.